resources/locale/en.yml

```diff
@@ -70,8 +70,8 @@ fof-byobu:
             view_private_discussions-when-flagged: View private discussions of other users if flagged
 
         settings:
-            use_tag_slug: Specify the which tag that all private discussions should use
-            use_tag_slug_help: Leave blank to allow users to select their own tag, applies to newly created private discussions only. Only primary tags are supported
+            use_tag_slug: Specify the tag that all private discussions must use
+            use_tag_slug_help: Leave blank to allow users to select their own tag. This applies to newly created private discussions only.
```

Comment:
This translation should probably be modified to reflect an admin now has to select 'no restriction', rather than 'leave blank'

Reply:
Changed. Now I think of it, no restriction should probably be a translation 🤦

Reply:
Now you mention it, yes it should, please